### PR TITLE
Fix import/export no additional file

### PR DESCRIPTION
**Pull request by dblark**
qwq

### CODE REVIEW

**Comment by Menci** [comment]
You exported the additional file id but didn't use it when importing. Maybe we just need a boolean to check if it has an additional file?

**Comment by dblark** [comment]
@Menci fix

File: modules/problem.js [hunk 1]
```diff
@@ -257,7 +257,7 @@ app.get('/problem/:id/export', async (req, res) => {
       limit_and_hint: problem.limit_and_hint,
       time_limit: problem.time_limit,
       memory_limit: problem.memory_limit,
-      additional_file_id: problem.additional_file_id,
+      have_additional_file_id: problem.additional_file_id !== null,
```
Comment (Menci) [inline modules/problem.js]: No `_id` here: `have_additional_file`.

Comment (dblark) [reply modules/problem.js]: I change a name for it.

**Pull request merged**